meta/statement_executor.go

```diff
@@ -281,6 +286,45 @@ func (e *StatementExecutor) executeShowContinuousQueriesStatement(stmt *influxql
 	return &influxql.Result{Series: rows}
 }
 
+func (e *StatementExecutor) executeShowShardsStatement(stmt *influxql.ShowShardsStatement) *influxql.Result {
+	dis, err := e.Store.Databases()
+	if err != nil {
+		return &influxql.Result{Err: err}
+	}
+
+	rows := []*influxql.Row{}
+	for _, di := range dis {
```

Comment:
I've got one on the `dmq-show-measurements2` branch, which is ready to merge.  @benbjohnson  can merge this as-is and I can convert this to use the one in `meta` package while doing other DMQ work.